modules/travel_pay/app/controllers/travel_pay/pings_controller.rb

```diff
@@ -2,7 +2,7 @@
 
 module TravelPay
   class PingsController < ApplicationController
-    before_action :authorize, only: [:authorized_ping]
+    skip_before_action :authenticate, only: :ping
```

Comment:
What are your thoughts on this? Mine are that the ping endpoint is unauthorized by the Travel Pay API, but shouldn't necessarily be unauthenticated by vets-api.

Reply:
Ok, I can get behind that. Wasn't trying to say it was wrong, just had a different thought. These ping endpoints are going away eventually, so no strong opinion on this.